openassessment/xblock/waffle_mixin.py

```diff
@@ -66,6 +68,7 @@ def is_feature_enabled(self, flag):
 
         return False
 
+    @cached_property
```

Comment:
What is the benefit of making it a cached property?

Reply:
waffle switches are _really_ slowly changing, so if we've looked up the value of a switch and flag once during the lifetime of an ORA block, it shouldn't be necessary to do it again.